Automattic/woocommerce-payments
# Added missing API docs sidebar links

#### Changes proposed in this Pull Request
Added missing API docs link in sidebar for https://github.com/Automattic/woocommerce-payments/pull/7384 and https://github.com/Automattic/woocommerce-payments/pull/7369





#### Testing instructions





* Make sure to see links for payment methods and payment intent.

-------------------


**Post merge**
